.golangci.yml

```diff
@@ -101,19 +101,81 @@ issues:
         - dogsled
       path: pkg/api/(.+)_test\.go # temporally disable dogsled in api test files
```

Comment:
`.golangci.yml` file has greatly changed on master branch (just heads up so we don't merge anything unintentional to master)